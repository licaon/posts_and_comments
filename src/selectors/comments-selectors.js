export const getComments = state => state.CommentsReducer;
export const getCommentsState = state => state.CommentsStateReducer;
