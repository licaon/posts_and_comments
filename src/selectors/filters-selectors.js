export const getSearchFiltersValue = state => state.SearchReducer;
