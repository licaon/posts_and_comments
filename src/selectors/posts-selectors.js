import { createSelector } from 'reselect';

export const getPosts = state => state.PostsReducer.posts;

export const getFilteredPosts = createSelector(
  [getPosts],
  posts => posts,
);
