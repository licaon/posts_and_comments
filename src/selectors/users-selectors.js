export const getUsers = state => state.UsersReducer.users;
