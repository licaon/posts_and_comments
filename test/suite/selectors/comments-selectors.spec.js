import * as CommentsSelectors from 'src/selectors/comments-selectors';

describe('CommentsSelectors', () => {
  it(`should execute getComments correctly`, (done) => {
    expect(CommentsSelectors.getComments({
      CommentsReducer: 'CommentsReducer',
    })).toMatchSnapshot();

    done();
  });
  it(`should execute getCommentsState correctly`, (done) => {
    expect(CommentsSelectors.getCommentsState({
      CommentsStateReducer: 'CommentsStateReducer',
    })).toMatchSnapshot();

    done();
  });
});
