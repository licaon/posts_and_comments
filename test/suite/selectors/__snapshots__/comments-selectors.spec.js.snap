// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsSelectors should execute getComments correctly 1`] = `"CommentsReducer"`;

exports[`CommentsSelectors should execute getCommentsState correctly 1`] = `"CommentsStateReducer"`;
